index.js

```diff
@@ -0,0 +1,41 @@
+document.addEventListener('DOMContentLoaded', function () {
+  let slides = document.querySelectorAll('.js-Earnings-left-side');
+  let indicators = document.querySelectorAll('.js-indicator');
```

Comment:
De nuevo, si no vas a modificar una variable hay que usar `const`
```suggestion
  const slides = document.querySelectorAll('.js-Earnings-left-side');
  const indicators = document.querySelectorAll('.js-indicator');
```